### ISSUE TITLE
[2020 CVPR] CNN-generated images are surprisingly easy to spot... for now

### ISSUE THREAD

**Issue by Jasonlee1995** [Jasonlee1995/AI_Papers#25]
해당 논문의 contributions 3가지, missing points 3가지를 언급하고 contribution 위주로 설명

- 3 Contributions
    * Forensics models train on CNN-generated images exhibit a surprising amount of generalization to other CNN synthesis methods
    * New dataset and evaluation metric for detecting CNN-generated images
    * Analyze the factors that account for cross-model generalization
- 3 Missing Points
    * Still suffer from trade-off between true detection and false-positive rates
    * Still weak from aggressive transformation (e.g. compression, resizing, re-sampling, ...)
    * Not work on classic shallow methods (e.g. Photoshop) 

### 1.1. Dataset
![image](https://user-images.githubusercontent.com/49643709/121351517-d443ea80-c966-11eb-86ee-9da4ee116bbd.png)

- 위와 같이 총 11개의 모델들에 대하여 dataset을 구축함
(check *Appendix B.1.* for more details)

### 1.2. Dataset Setting
- Default resolution : 256 * 256
- Lower resolution : bilinear interpolation을 통해 shorter side를 256으로 맞춤 (aspect ratio는 유지)
- Higher resolution : 그대로 둠

### 1.3. Data Augmentation
- Train default augmentation : random horizontal flip과 224 * 224 random crop
- Extra train augmentations
    * No aug : no extra augmentation
    * Gaussian blur : random blur on image with sigma from Uniform[0, 3] before crop (0.5 prob)
    * JPEG : randomly JPEG the image with quality from Uniform{30, 31, ..., 100} (0.5 prob)
    * Blur+JPEG (0.5) : image is randomly blurred and JPEG-ed (0.5 prob)
    * Blur+JPEG (0.1) : image is randomly blurred and JPEG-ed (0.1 prob)
- Test augmentation : 224 * 224 center crop

### 2.1. Model
- ImageNet pre-trained ResNet-50을 사용하며, binary classification setting 적용 (true or false)

### 2.2. Train Objective
- 1개의 real/fake dataset으로 학습하여 나머지 10개의 real/fake dataset을 잘 맞추는게 목표

### 2.3. Train
- Adam optimizer with (beta1=0.9, beta2=0.999), batch size 64, lr 1e-4 with lr scheduler로 학습 진행
(check *Appendix B.2.* for more details)

### 2.4. Evaluation
- AP (average precision)
(check *Appendix A.6.* for more details)

### 3.1. Effect of Data Augmentation
![image](https://user-images.githubusercontent.com/49643709/121355949-5e8e4d80-c96b-11eb-9555-1bf647427be0.png)

저자들은 data augmentation이 어떤 영향을 미치는지 실험을 진행함

### 3.3?. Choice of Train Dataset
![image](https://user-images.githubusercontent.com/49643709/121353634-048c8880-c969-11eb-9d31-ec6e04e89d46.png)

저자들은 어떠한 dataset이 train dataset으로 사용하면 좋을지에 대하여 실험을 진행함

ProGAN, BigGAN, DIP에 대하여 실험함
(check *Appendix A.3.* and *Appendix A.4.* for more details)

저자들은 위 실험 결과에 대하여 분석한 결과는 다음과 같음

- ProGAN으로 학습한 모델이 BigGAN으로 학습한 모델보다 성능이 좋은 이유
    * ProGAN model was trained on ensemble (one model per class), BigGAN images were generated with a single model
- DIP로 학습한 모델이 다양한 dataset에서 working함
    * 다양한 모델들이 low-level artifacts를 share함
    * 하지만 low-level artifacts만으로 판단하기에는 부족함
